README.md

```diff
@@ -21,3 +21,15 @@ Please read the [contribution guidelines](CONTRIBUTING.md).
 
 ### Front End
 * [Pug](https://github.com/pugjs/pug)
+
+### Testing
+* Testing Framework: [Mocha](http://mochajs.org/)
+* Assertion Library: [Chai](http://chaijs.com/)
+* Test Spies, Stubs and Mocks: [Sinnon](http://sinonjs.org/)
+* Test Coverage: [Istanbul](https://github.com/gotwarlost/istanbul)
+* ?? Test Runner: [Karma](https://karma-runner.github.io/1.0/index.html)
+
+     #### Testing Learning & Resources
```

Comment:
Let's move this section into another file called `RESOURCES.md`, and provide a link in this README so that the README doesn't get too unwieldy.